sections/semantics-embedded-content.include

```diff
@@ -8310,16 +8219,6 @@ zero or more <{track}> elements, then
   been removed; only once a media element is in a state where no further audio could ever be played
   by that element may the element be garbage collected.
 
-  <p class="note">
-    It is possible for an element to which no explicit references exist to play audio,
-  even if such an element is not still actively playing: for instance, it could have a <a>current
-  media controller</a> that still has references and can still be unpaused, or it could be
-  unpaused but stalled waiting for content to buffer, or it could be still buffering, but with a
```

Comment:
'suspend' is not part of media controller. This note is still very much valid. Just do a more surgical removal. ;)